### DISCUSSION TITLE
Approximately after how many releases, are we going to see the first stable release?

### DISCUSSION THREAD

**Discussion by ghost** [DioxusLabs/dioxus#2901] [General]
I know dioxus for a long time and I saw all those api breaking changes it went through. I always wanted to learn a ui framework, since its not stable I have a little anxiety of starting a personal projects. So, can we know approximately when the first stable release will be dropped?

**Comment by xpe** [comment]
@sumit-modak Given that you are considering Dioxus for personal projects -- presumably ones that don't involve significant business risk -- I wouldn't put much emphasis on when Dioxus 1.0 is released. I would start your prototyping now. Why not try Dioxus and compare it against your other top contenders? Take notes about what you learn. Share it back with the team.

More generally, to the extent anyone sees the value in WASM and Rust, why do you care all that much about waiting for 1.0?
1. The Web is always changing. 
2. Code-assistance tools are getting better and better.

So, why wait?

I'm not "on the inside", but _I wouldn't expect_ Dioxus to have an answer to this question yet. They probably have some goals and targets in mind, but I don't have any information on that.

If I were to make a broad guess (p=~80%) about when a 1.0 release happens, I would think it would probably happen before July 2026.

However, the criteria for 1.0 for any software project often has to do with factors that are not under direct control of a team, such as:
- user feedback
- business climate / competition
- parallel changes / innovations in the ecosystem (in this case, WASM and Rust being particularly important)
